lte/gateway/python/integ_tests/federated_tests/docker/docker-compose.override.yml

```diff
@@ -16,10 +16,10 @@ services:
 
   control_proxy:
     extra_hosts:
-      - controller.magma.test:127.0.0.1
-      - bootstrapper-controller.magma.test:127.0.0.1
-      #- controller.magma.test:10.0.2.2
-      #- bootstrapper-controller.magma.test:10.0.0.2
+      #- controller.magma.test:127.0.0.1
```

Comment:
What's the purpose of the commented config here? Can we add a comment explaining the options of using localhost vs 10.0.0.2?

Reply:
10.0.0.2 is used when FEG is run locally in the host. 127.0.0.1 when run on Vagrant. 
I can add a note there. But I would rather prefer to leave that comments because the knowledge of those routes is not easy to figure out. Leave them commented will provide context in case we want to switch back the test environment to the host.


Note that config file is just a test file exclusively for federated integ test.